test/integration/hotrestart_test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
```

Comment:
Instead of copying this code (not sure what changed vs. the other file), can we potentially just call this script from the other place to avoid duplication/cruft?